installer/scripts/quay-helper.sh

```diff
@@ -204,40 +204,6 @@ quay_create_secret() {
     return 1
 }
 
-# Create a repository in organization with the name informed via environment,
```

Comment:
I think you can also remove the definition of the `QUAY_REPOSITORY` var.

Reply:
Fixed. Thank you for the reminder